### PR TITLE
feat: Make admin url and login from

### LINKED ISSUE #105: FeedAHelp: Create a secure admin URL with a simple login form only
In order for Admin to login to FeedAHelp, We need a Special URL with a simple login form as follows

Image

This ticket has nothing to do with #98, this is only for admin

### PR DESCRIPTION
**Pull request by biplob2358**
# Description
Create a secure admin URL with a simple login form only

To view type `/admin`

Fixes #105 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Closes #105 
image

### CODE REVIEW

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+`;
+
+const RotatedBackground = styled.div`
+  position: absolute;
+  content: "";
+  width: 300px;
+  height: 300px;
+  background-color: rgb(209, 193, 255);
+  transform: rotate(45deg);
+  left: -180px;
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.styled.tsx]: convert everything to `rem`

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <Styled.Heading>Admin Login</Styled.Heading>
+        <Styled.InputContainer>
+          <Styled.InputIcon
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            fill="#2e2e2e"
+            viewBox="0 0 16 16"
+          >
+            <path d="M13.106 7.222c0-2.967-2.249-5.032-5.482-5.032-3.35 0-5.646 2.318-5.646 5.702 0 3.493 2.235 5.708 5.762 5.708.862 0 1.689-.123 2.304-.335v-.862c-.43.199-1.354.328-2.29.328-2.926 0-4.813-1.88-4.813-4.798 0-2.844 1.921-4.881 4.594-4.881 2.735 0 4.608 1.688 4.608 4.156 0 1.682-.554 2.769-1.416 2.769-.492 0-.772-.28-.772-.76V5.206H8.923v.834h-.11c-.266-.595-.881-.964-1.6-.964-1.4 0-2.378 1.162-2.378 2.823 0 1.737.957 2.906 2.379 2.906.8 0 1.415-.39 1.709-1.087h.11c.081.67.703 1.148 1.503 1.148 1.572 0 2.57-1.415 2.57-3.643zm-7.177.704c0-1.197.54-1.907 1.456-1.907.93 0 1.524.738 1.524 1.907S8.308 9.84 7.371 9.84c-.895 0-1.442-.725-1.442-1.914z"></path>
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx]: This an other will make this page very dirty, clean it up, move it to external file

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        </Styled.InputContainer>
+        <Styled.InputContainer>
+          <Styled.InputIcon
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
+            fill="#2e2e2e"
+            viewBox="0 0 16 16"
+          >
+            <path d="M8 1a2 2 0 0 1 2 2v4H6V3a2 2 0 0 1 2-2zm3 6V3a3 3 0 0 0-6 0v4a2 2 0 0 0-2 2v5a2 2 0 0 0 2 2h6a2 2 0 0 0 2-2V9a2 2 0 0 0-2-2z"></path>
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx]: same here, its make the page messy

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
do the coding changes and design change
image

**Review by urmirahman — DISMISSED** [review]
.

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.styled.tsx (hunk 1 above)
Comment (biplob2358) [reply apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.styled.tsx]: resolved

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx (hunk 2 above)
Comment (biplob2358) [reply apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx]: resolved

File: apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx (hunk 3 above)
Comment (biplob2358) [reply apps/feedahelp/src/components/ui/features/adminLogin/AdminLogin.tsx]: resolved

**Review by TanvirAlam — APPROVED** [review]
LGTM, lets merge and fix it later when we have DB in

**Review by urmirahman — APPROVED** [review]
LGTM

**Pull request merged**